packages/rocketchat-lib/lib/RoomTypeConfig.js

```diff
@@ -242,4 +242,15 @@ export class RoomTypeConfig {
 
 		return { title, text };
 	}
+
+	/**
+	 * Check if there is a user with the same id and loginToken
+	 * @param {object} allowData
+	 * @return {object} User's object from db
+	 */
+	allowFileUpload(allowData) {
+		const { rc_uid, rc_token } = allowData;
+		return rc_uid && rc_token && RocketChat.models.Users.findOneByIdAndLoginToken(rc_uid, rc_token);
```

Comment:
since this is being [done already](https://github.com/RocketChat/Rocket.Chat/pull/10514/files#diff-5239c86106ca820cedec036b79577f85R216) previously, I think this default function can return `false` only.

Reply:
Fixed.